### PR TITLE
Synchronize mpp_execute option description and precedence rules in en…

### PR DESCRIPTION
**Pull request by dyozie**
Updates/adds mpp_execute documentation in reference page, to match sgml changes from https://github.com/greenplum-db/gpdb/pull/6505

### CODE REVIEW

**Comment by mkiyama** [comment]
@dyozie 

It seems that FOREIGN DATA WRAPPER, FOREIGN TABLE, and SERVER are all linked.
Would it make sense to specify the order of precedence in each of these?

Also, probably beyond the scope of this PR. USER MAPPING is only listed as a link in CREATE SERVER. Should there be a short description in notes?

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_DATA_WRAPPER.xml [hunk 1]
```diff
@@ -47,6 +47,23 @@
...
                 </plentry>
+                <plentry>
+                    <pt>mpp_execute { 'master | 'any' | 'all segments' }</pt>
+                    <pd>
+                        <p>An option that identifies the host from which the foreign data-wrapper
+                            requests data: <ul id="ul_zcg_2vd_mgb">
+                                <li><codeph>master</codeph> (the default)—Request data from the
+                                    master host.</li>
+                                <li><codeph>any</codeph>—Request data from either the master host or
+                                    one any segment, depending on which path costs less.</li>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_DATA_WRAPPER.xml]: one any -> any one ?

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml [hunk 2]
```diff
@@ -98,11 +98,21 @@
...
-                        master host, any (master or single segment) host, or all segment hosts. The default option value
-                       is '<codeph>master</codeph>'.</p>
-                      <p>Use of the foreign table <codeph>mpp_execute</codeph> option, and the
-                       specific modes supported, is foreign data-wrapper-specific.</p>
+                        <p>An option that identifies the host from which the foreign data-wrapper
+                            requests data: <ul id="ul_zcg_2vd_mgb">
+                                <li><codeph>master</codeph> (the default)—Request data from the
+                                    master host.</li>
+                                <li><codeph>any</codeph>—Request data from either the master host or
+                                    one any segment, depending on which path costs less.</li>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml]: same as above

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml [hunk 3]
```diff
@@ -98,11 +98,23 @@
...
+                                <li><codeph>all segments</codeph>—Request data from all segments.
+                                    With this option value, the foreign data-wrapper must have a
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml]: what about "To support this option value, ..."?  the foreign data-wrapper should document how it uses the mpp_execute option, and i assume that it is possible that the FDW does not support this value.

(same comment on the other two ref pages.)

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml [hunk 4]
```diff
@@ -98,11 +98,23 @@
...
+                                    policy that matches the segments to data.</li>
+                            </ul></p>
+                        <p>Use of the foreign table <codeph>mpp_execute</codeph> option, and the
+                            specific modes supported, is foreign data-wrapper-specific.</p>
+                        <p>The <codeph>mpp_execute</codeph> option can be configured in multiple
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_FOREIGN_TABLE.xml]: "configured" -> "specified" ?

**Pull request merged**